teps/README.md

```diff
@@ -262,4 +262,5 @@ This is the complete list of Tekton teps:
 |[TEP-0112](0112-replace-volumes-with-workspaces.md) | Replace Volumes with Workspaces | proposed | 2022-07-20 |
 |[TEP-0114](0114-custom-tasks-beta.md) | Custom Tasks Beta | implementable | 2022-07-12 |
 |[TEP-0116](0116-referencing-finally-task-results-in-pipeline-results.md) | Referencing Finally Task Results in Pipeline Results | implemented | 2022-08-11 |
+|[TEP-0117](0117-tekton-results-logs.md) | Tekton Results Logs | proposed | 2022-07-19 |
```

Comment:
fyi, the lint job is failing because the date here doesn't match the one in the TEP. Running `teps/tools/teps.py table` should fix that.